srcpkgs/nootka/template

```diff
@@ -1,14 +1,19 @@
 # Template file for 'nootka'
 pkgname=nootka
-version=1.4.7
-revision=2
+version=2.0.0
+revision=1
 wrksrc="nootka-${version}-source"
 build_style=cmake
 hostmakedepends="qt5-qmake qt5-host-tools"
-makedepends="alsa-lib-devel fftw-devel libvorbis-devel qt5-devel soundtouch-devel"
+makedepends="alsa-lib-devel fftw-devel jack-devel libvorbis-devel qt5-devel
+ qt5-declarative-devel qt5-quickcontrols2-devel qt5-tools-devel soundtouch-devel
+ $(vopt_if pulseaudio pulseaudio-devel)"
```

Comment:
Any reason not to enable by default? Would you like to have it off for your own usage?

Reply:
The previous version 1.4.7 was built without pulseaudio (though it is supported by that version). So I just added it as an option if someone might want to have it. Personally I don’t use pulseaudio and nootka works just fine. But I’m not opposed to enabling it by default as many packages pull in libpulseaudio anyway. ;)

Reply:
I think it makes sense to have it enabled by default, a bit more user friendly :)

Reply:
Done. :)